fetools-app/src/pages/Tables.jsx

```diff
@@ -64,45 +70,43 @@ const Table = () => {
         title="Code Snippets"
         icon="integration_instructions"
       >
-        <TabSwitcher title="" buttons={['css', 'tailwind']}>
-          <CodeBlock title="" code={''} lang="css" />
+        {/* <CodeGeneratorTable toastState={toastState} tableConfig={tableConfig}  /> */}
+
+        <TabSwitcher buttons={["HTML & CSS", "Tailwind"]}>
+          <CodeBlock
+            toastState={toastState}
+            code={generateTableCode(tableConfig)}
+            lang="html" 
+          />
+
           <CodeBlock
-            title=""
-            code={` /* Add this to your tailwind.config.js file */
-            module.exports = {
-              theme: {
-                extend: {
-                  boxShadow: {
-                    '3xl': ,
-                  }
-                }
-              }
-            }
-          `}
-            lang="css"
+            toastState={toastState}
+            code={generateTableCode(tableConfig, true)}
+            lang="html" 
           />
         </TabSwitcher>
       </ToolSection>
 
       <GoDeeper
         linksData={[
           {
-            url: 'https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow',
+            url: "https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow",
             textValue:
-              'box-shadow from MDN - https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow ',
+              "box-shadow from MDN - https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow ",
           },
           {
-            url: 'https://www.w3schools.com/cssref/css3_pr_box-shadow.php',
+            url: "https://www.w3schools.com/cssref/css3_pr_box-shadow.php",
             textValue:
-              'box-shadow from W3 School - https://www.w3schools.com/cssref/css3_pr_box-shadow.php ',
+              "box-shadow from W3 School - https://www.w3schools.com/cssref/css3_pr_box-shadow.php ",
           },
           {
-            url: 'https://css-tricks.com/almanac/properties/b/box-shadow/',
+            url: "https://css-tricks.com/almanac/properties/b/box-shadow/",
             textValue:
-              'box-shadow from Css-tricks - https://www.w3schools.com/cssref/css3_pr_box-shadow.php ',
+              "box-shadow from Css-tricks - https://www.w3schools.com/cssref/css3_pr_box-shadow.php ",
           },
         ]}
       />
+      <Toast toastState={toastState} />
```

Comment:
We're going to have to do something inside the toast block to truncate the code that is displayed in the toast.